report_generator/exporter/data/sponsor.html

```diff
@@ -147,7 +164,7 @@ <h2>Proposed Topics</h2>
                 </div>
             </div><!-- /content -->
             <div data-role="footer" data-position="fixed" data-tap-toggle="false" class="jqm-footer">
-                <p>Created by PyCon TW 2019 Sponsorship Team</p>
+                <p>Created by PyCon TW {{ jinja2_tag_year }} Sponsorship Team</p>
```

Comment:
I'm woundering why you use the prefix `jinja2_tag` here.

Reply:
For better searching result. It will prevent me to update any tag that I forgot to render, and it will be very ugly on the report if I forgot to render any tag and did not catch it when manually reviewing the review content.

Reply:
In this case, should we add this prefix to all jinja tags?

Reply:
Yes from software engineering point of view. However, I will leave the unification to another pull request instead. The renaming will be a huge refactoring for legacy code, and only make sure the prefix is applied in this batch.